Formula/libxcb.rb

```diff
@@ -3,12 +3,12 @@ class Libxcb < Formula
   homepage "https://www.x.org/" ### http://www.linuxfromscratch.org/blfs/view/svn/x/x7lib.html
   url "https://xcb.freedesktop.org/dist/libxcb-1.13.1.tar.bz2"
   sha256 "a89fb7af7a11f43d2ce84a844a4b38df688c092bf4b67683aef179cdf2a647c4"
+  revision 1
   # tag "linuxbrew"
 
   bottle do
     root_url "https://linuxbrew.bintray.com/bottles-xorg"
     cellar :any_skip_relocation
-    sha256 "657c38f163f14bc65729f6b15414da87d896adc3d005642289bd9d4c0544b05f" => :x86_64_linux
```

Comment:
It's not necessary to delete this line in this repo (we don't use `--keep-old` flag here).